js/todo-list.js

```diff
@@ -0,0 +1,45 @@
+function makeTodo(todo) {
+    const html = /*html*/`
+        <li class="todo">
+            <h3 class="todo ${todo.date} < Date.now()? 'overdue' : ''}">
```

Comment:
```suggestion
            <h3 class="todo ${new Date(todo.date) < Date.now() ? 'overdue' : ''}">
```
Typo: You had an extra } that wasn't allowing all the Javascript through. Also, the date was being returned as a string on local storage load, so this was just a quick way to make sure it was back in to Date format (to properly compare it in if statement)